### PR TITLE
Fix export configurations

### PR DESCRIPTION
**Pull request by alexisjanvier**

### CODE REVIEW

File: js/categories/config.js [hunk 1]
```diff
@@ -5,7 +5,10 @@ export default function (nga, admin) {
         .fields([
             nga.field('name'),
         ])
-        .listActions(['<ma-filtered-list-button entity-name="products" filter="{ category_id: entry.values.id }" size="xs" label="Related products"></ma-filtered-list-button>', 'edit', 'delete']);
+        .listActions(['<ma-filtered-list-button entity-name="products" filter="{ category_id: entry.values.id }" size="xs" label="Related products"></ma-filtered-list-button>', 'edit', 'delete'])
+        .exportFields([
+            nga.field('name').label('Name'),
```
Comment (fzaninotto) [inline js/categories/config.js]: You shouldn't need that, the export should use listFields by default.

File: js/reviews/config.js [hunk 2]
```diff
@@ -77,7 +77,8 @@ export default function (nga, admin) {
             '<batch-approve type="accept" selection="selection"></batch-approve>',
             '<batch-approve type="reject" selection="selection"></batch-approve>',
             'delete'
-        ]);
+        ])
+        .actions(['filter']);
```
Comment (fzaninotto) [inline js/reviews/config.js]: why remove `delete`?

File: js/categories/config.js (hunk 1 above)
Comment (alexisjanvier) [inline js/categories/config.js]: There's a bug in export for fields with `template` type. This PR is just a fix, waiting a patch from ng-admin.

**Comment by alexisjanvier** [comment]
https://github.com/marmelab/ng-admin/pull/949 fixes the problem. I close the PR

**Pull request closed without merging**